app/views/profiles/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<section id="profile">
+  <profile
+    api-key="<%= FirebaseConfig.api_key %>"
+    project-id="<%= FirebaseConfig.project_id %>"
+    db-user-id="<%= current_user.id or '' %>">
```

Comment:
[nitpick] To help with the rendering, could you use the `<%= … -%>` syntax, to avoid rendering a line break at the end of the expression.

Reply:
done